tests/robot-tests/README.md

```diff
@@ -59,137 +58,139 @@ If you intend to run the tests from your local machine, you will also need to cr
-
-### IDE
+## Test data:
 
-If searching for an IDE to add/edit these tests, consider using IntelliJ with the following plugins:
+After a group discussion, it was decided that tests will, as far as is possible, create their own test data. This means that any tests you write that alter data, you will need to create that data from scratch.
 
-- IntelliBot 
-- Robot Plugin 
+## **IDE**
 
-This should give you autocompletion and allow you to click through to keywords defined in both `.robot` and `.py` files. For this to work, you'll need to change the Project Structure to use "No SDK".
+If searching for an IDE to add/edit these tests, consider using IntelliJ, Pycharm or VScode with the following extensions: 
 
-IntelliJ also allows you use to External Tools to right click on a file and run that file exclusively with these settings:
-- Program: `/home/${USER}/.local/bin/pipenv` OR `C:\Python38\Scripts\pipenv.exe` OR wherever pipenv is located -- use `whereis pipenv` on linux or `where pipenv` on windows
-- Arguments: `run python run_tests.py --visual -i robot -f "$FilePath$" -e dev`
-- Working directory: `$ProjectFileDir$` (which should represent the robot-tests directory. You may have to do something like `$ProjectFileDir$/../tests/robot-tests`)
+IntelliJ / Pycharm: 
+- [IntelliBot](https://plugins.jetbrains.com/plugin/7386-intellibot) 
+- [Robot Plugin](https://plugins.jetbrains.com/plugin/7430-robot-plugin)
 
+VScode:
+- [Robot Framework Intellisense](https://marketplace.visualstudio.com/items?itemName=TomiTurtiainen.rf-intellisense)
 
-Alternatively you can use Pycharm CE (https://www.jetbrains.com/pycharm/download/#section=windows) as an IDE and follow the above steps to enable External Tools. 
+### **Additional IntelliJ settings**
+  This should give you autocompletion and allow you to click through to keywords defined in both `.robot` and `.py` files. For this to work in, you'll need to change the Project Structure to use "No SDK".
 
-In addition to this you can also use VScode and use the "Robot framework intellisense" extension. 
+  IntelliJ also allows you use to External Tools to right click on a file and run that file exclusively with these settings:
+  - Program: `/home/${USER}/.local/bin/pipenv` OR `C:\Python38\Scripts\pipenv.exe` OR wherever pipenv is located -- use `whereis pipenv` on linux or `where pipenv` on windows
+  - Arguments: `run python run_tests.py --visual -i robot -f "$FilePath$" -e dev`
+  - Working directory: `$ProjectFileDir$` (which should represent the robot-tests directory. You may have to do something like `$ProjectFileDir$/../tests/robot-tests`)
 
-## Troubleshooting
+## **Troubleshooting**
 
 ### The tests are flaky when I run them locally against the public frontend.
 Try running the frontend with `npm run build & npm run start:prod`.
 
+### Test fails after not finding an element after x amount of seconds.
```

Comment:
Maybe also mention the `--print-keywords` option for seeing a detailed breakdown of the currently executing keywords, which also shows failing keywords immediately.